pkg/controller/storagecluster/reconcile.go

```diff
@@ -911,6 +913,7 @@ else
   echo "OSD ${FAILED_OSD_ID} is down. Proceeding to mark out and purge"
   ceph osd out osd.${FAILED_OSD_ID} 
   ceph osd purge osd.${FAILED_OSD_ID} --force --yes-i-really-mean-it
+  ceph osd crush rm $HOST_TO_REMOVE
```

Comment:
Will errors be ignored for this command, right? We just need to confirm the job succeeds even if this fails.

Reply:
Yeah. The job should succeed. The command will remove host successfully if none osd is present under the host if osd is present, the host won't be removed, since it's not empty. The command will throw such a message.

Reply:
For eg: ```[root@rhhi-cap2m /]# ceph osd crush rm  rhhi-cap3m-lab-eng-blr-redhat-com
Error ENOTEMPTY: (39) Directory not empty```